.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
     unzip -o terraform_${TF_VERSION}_linux_amd64.zip -d ~/bin
     rm terraform_${TF_VERSION}_linux_amd64.zip
   - pip install --user yamllint
-  - eval "$(gimme 1.6)"
+  - eval "$(gimme 1.8)"
```

Comment:
Lint errors occurred using 1.6, in relation to an import. Upgrading to 1.8 fixed these.